prime-router/settings/STLTs/WA/wa-phd.yml

```diff
@@ -0,0 +1,127 @@
+- name: "wa-phd"
+  description: "Washington Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "WA"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "wa-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/WA/WA-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "WADOHPHRED^2.16.840.1.113883.3.237.4.2^ISO"
+        receivingApplicationOID: "2.16.840.1.113883.3.237.4.2"
+        receivingFacilityName: "dn1fro00^1.3.6.1.4.1.38630.2.1.1.19^ISO"
+        receivingFacilityOID: "1.3.6.1.4.1.38630.2.1.1.19"
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: null
+        reportingFacilityName: "7uycso49"
+        reportingFacilityId: "1.3.6.1.4.1.38630.2.1.1.519"
+        reportingFacilityIdType: "ISO"
+        suppressQstForAoe: true
+        suppressHl7Fields: "PID-39-1, PID-39-2, PID-39-3"
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "ONLY_DIGITS_IN_COMPONENT_ONE"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "ORGANIZATION_NAME"
+        valueSetOverrides: {}
+        nameFormat: "washington"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: true
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: true
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'WA') or (Bundle.entry.resource.ofType(Patient).address.state = 'WA')"
+      qualityFilter:
+              # "doesNotMatch(patient_city,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).where(address.city = 'UNKNOWN')"
```

Comment:
We can do something like   .where(address.city.lower() = 'unknown') to replace line 63, 64, 65.  And do the same for patient_first_name, last_name, family, state.  Note, the .lower() will do it.